### PR TITLE
Set priority on listeners to ensure they are called in proper order

### PR DESCRIPTION
**Pull request by szymach**

### CODE REVIEW

File: Resources/config/services.xml [hunk 1]
```diff
@@ -65,18 +65,18 @@
         <!-- Listeners -->
         <service id="fsi_doctrine_extensions.listener.uploadable_post_load" class="%fsi_doctrine_extensions.listener.uploadable_post_load.class%">
             <argument type="service" id="fsi_doctrine_extensions.listener.uploadable" />
-            <tag name="fsi_doctrine_extensions.listener.uploadable" />
+            <tag name="fsi_doctrine_extensions.listener.uploadable" priority="2" />
```
Comment (JarekW) [inline Resources/config/services.xml]: Can we place this priorities more widely? We can later put events in end application in between those events. Symfony has similar approach: http://symfony.com/doc/current/reference/events.html#kernel-request

File: spec/FSi/Bundle/DoctrineExtensionsBundle/DependencyInjection/FSIDoctrineExtensionsExtensionSpec.php [hunk 2]
```diff
@@ -59,8 +59,13 @@ function it_should_add_tag_to_uploadable_listener_service(
         $builder->setParameter('fsi_doctrine_extensions.default.filesystem.base_url', '/uploaded')
             ->shouldBeCalled();
 
+        $tag = array(array('priority' => 0));
```
Comment (JarekW) [inline spec/FSi/Bundle/DoctrineExtensionsBundle/DependencyInjection/FSIDoctrineExtensionsExtensionSpec.php]: []

Comment (szymach) [inline spec/FSi/Bundle/DoctrineExtensionsBundle/DependencyInjection/FSIDoctrineExtensionsExtensionSpec.php]: I will convert all `array()` to shorthand in a separate PR.

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Pull request merged**